validators/build.gradle

```diff
@@ -21,7 +21,6 @@ dependencies {
 }
 
 jar {
-    enabled = true
```

Comment:
we can revert all these changes back to normal if above solution works, or remove this completely